tools/bag_processing/scripts/rosbag_fix_all.py

```diff
@@ -99,12 +158,37 @@ class CustomFormatter(argparse.ArgumentDefaultsHelpFormatter):
         default=False,
         action="store_true",
     )
+    parser.add_argument(
+        "--debayer",
+        help="debayer the nav or dock cam",
+        default=False,
+        action="store_true",
+    )
+    parser.add_argument(
+        "--decode-haz",
+        default="",
+        help="decode the extended haz cam topic into points and amplitude_int, argument is robot name",
+        type=str,
+    )
+    parser.add_argument(
+        "--filter",
+        help="filter the bagfile with only some topics",
```

Comment:
It's not obvious what format this argument takes and hard to track down even from the source code. I think I figured out you can give it arbitrary options for `rosbag_filter_topics.py` but not 100% sure (and is there any guidance about quoting?). Probably good to clarify.